src/current/_includes/common/telemetry-table.html

```diff
@@ -0,0 +1,932 @@
+<table>
+ <tr>
+  <th>Category</th>
+  <th>Description</th>
+  <th>Fields</th>
+  <th>Example</th>
+ </tr>
+ <tr>
+  <td>Node information</td>
+  <td>Information about the CockroachDB node reporting diagnostics.</td>
+  <td>
+   <ul>
+    <li>Node ID</li>
+    <li>Bytes</li>
+    <li>Key Count</li>
+    <li>Range Count</li>
+    <li>Uptime</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"node": {
+        "nodeId": 12,
+        "bytes": "2270774884600",
+        "keyCount": "27008719525",
+        "rangeCount": "7015",
+        "uptime": "1418000"
+    }
+</pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Store information</td>
+  <td>Information about individual stores on the CockroachDB node.</td>
+  <td>
+   <ul>
+    <li>Node ID</li>
+    <li>Store ID</li>
+    <li>Bytes</li>
+    <li>Key Count</li>
+    <li>Range Count</li>
+    <li>Capacity</li>
+    <li>Available</li>
+    <li>Used</li>
+    <li>Encryption Algorithm</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"stores": [
+        {
+            "nodeId": 12,
+            "storeId": 39,
+            "bytes": "560732812950",
+            "keyCount": "6719249260",
+            "rangeCount": "1730",
+            "capacity": "395188764700",
+            "available": "167600533600",
+            "used": "212346898300",
+            "encryptionAlgorithm": "0"
+        }
+    ]
+</pre>
+  </td>
+ </tr>
+ <tr>
+  <td>SQL instance information</td>
+  <td>Information about the SQL instance reporting diagnostics.</td>
+  <td>
+   <ul>
+    <li>SQL Instance ID</li>
+    <li>Uptime</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"sql": {
+        "sqlInstanceId": 12,
+        "uptime": "1418000"
+    }
+  </pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Altered cluster settings</td>
+  <td>
+   <p>
+      Cluster settings that have been altered from their default values.
+    </p>
+   <p>
+      Note that any values of type ‘string’ are redacted, such as the cluster organization.
+    </p>
+  </td>
+  <td>
+   <ul>
+    <li>Setting Name</li>
+    <li>Setting Value</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
+"alteredSettings": {
+    "cluster.organization": "\u003credacted\u003e", "kv.lease.expiration_leases_only.enabled": "false",
+    "kv.rangefeed.enabled": "true"
+  }
+</pre>
+  </td>
+ </tr>
+ <tr>
+  <td>Zone Configurations</td>
+  <td>Zone configurations applied to tables or databases.</td>
+  <td>Zone ID
```

Comment:
Zone ID and Zone Configuration need to be in `<ul>` and `<li>` tags

Reply:
fixed, thanks for the catch!